modules/components/pages/outputs/couchbase.adoc

```diff
@@ -0,0 +1,272 @@
+= couchbase
+:type: output
+:status: experimental
```

Comment:
If experimental is now considered beta, we can replace this with `:page-beta: true`

Reply:
Updated as requested.